ipemu/src/TestBench.scala

```diff
@@ -35,11 +35,22 @@ class TestBench(generator: SerializableModuleGenerator[T1, T1Parameter]) extends
   chisel3.reflect.DataMirror.modulePorts(dut)
     .filter(p => !p._1.contains("Probe")).foreach(_._2 <> DontCare)
 
+  // TODO: refactor this with generics
+  val lsuProbes = dut.lsuProbes.zipWithIndex.map{case (p, idx) =>
+    val wire = Wire(p.cloneType).suggestName(s"lsu${idx}Probe")
+    wire := probe.read(p)
+  }
+
   val laneProbes = dut.laneProbes.zipWithIndex.map{case (p, idx) =>
     val wire = Wire(p.cloneType).suggestName(s"lane${idx}Probe")
     wire := probe.read(p)
   }
 
+  val laneVrfProbes = dut.laneVrfProbes.zipWithIndex.map{case (p, idx) =>
+    val wire = Wire(p.cloneType).suggestName(s"lane${idx}VrfProbe")
+    wire := probe.read(p)
+  }
```

Comment:
There are multiple VRF in lane, so `laneVrfProbes` should be:
```
Seq[Int, Seq[Int, VRFProbes]]
    ^ Lanes
             ^VRFBanks
```

Reply:
I'm sorry, could you tell me where we use VRFBanks to create vrfprobes? In the https://github.com/chipsalliance/t1/blob/6b243b00a85a080160193810f925648472c3833e/ipemu/src/VerificationModule.scala#L75-L93, there are only have one loop.

Reply:
I see, we use the data entering read/write queue. That's a reasonable solution. but in the future we should resolve this, since we also need to benchmark the vrf bank conflict result.